src/main/java/stream/flarebot/flarebot/commands/FlareBotManager.java

```diff
@@ -245,14 +252,32 @@ public void saveLocalisation() {
                     PreparedStatement statement = conn
                             .prepareStatement("INSERT INTO localisation (guild_id, locale) VALUES (?, ?) ON DUPLICATE KEY locale = VALUES(locale)");
                     statement.setString(1, s);
-                    statement.setString(2, locale.get(s));
+                    statement.setString(2, locale.get(s).toLanguageTag());
                     statement.execute();
                 });
             } catch (SQLException e) {
                 e.printStackTrace();
             }
         }
+        for (Config localeFile : localeFiles.values()) {
+            localeFile.save();
+        }
     }
 
+    public String getLang(Language lang, String id) {
+        if (locale.containsKey(id)) {
+            Locale l = locale.get(id);
+            String path = lang.name().toLowerCase().replaceAll("\\_", ".");
+            Config config = localeFiles.get(l);
+            return config.getString(path);
+        } else {
+            locale.put(id, Locale.UK);
```

Comment:
No. Do not do something like this. Default values shouldn't be inserted, but taken with getOrDefault, to save memory. Keep in mind, this is a reference, meaning it takes memory, even though its a singleton.